### PR TITLE
Remove unnecessary log output

### PR DESCRIPTION
**Pull request by namsonx**

### CODE REVIEW

**Review by test-fullautomation — CHANGES_REQUESTED** [review]
Hi Son,
Thank you for your pull-request.

I'm a bit confused.
What is the issue for this change?

check passed must be displayed if a min- or maximum version is provided and we pass the check.

check passed mustn't be displayed if both, min- and maximum version are not provided.

Should be like
```
if (self.sMinVersion.strip() != '' or self.sMaxVersion.strip() != '') and bVersionCheck:
   logger.info(.....)
```
Thank you,
 Thomas

File: RobotFramework_TestsuitesManagement/Config/CConfig.py [hunk 1]
```diff
@@ -779,9 +779,6 @@ def verifyRbfwVersion(self):
             self.versioncontrol_error('conflict_max', self.sMaxVersion, sCurrentVersion)
             bVersionCheck = False
 
-        if bVersionCheck:
```
Comment (HolQue) [inline RobotFramework_TestsuitesManagement/Config/CConfig.py]: Now `bVersionCheck` is set, but not used. This makes no sense.
Nevertheless: You removed a mandatory message. Why? A result of an executed version check has to be logged.

**Review by HolQue — CHANGES_REQUESTED** [review]
See my code comments ...

**Comment by namsonx** [comment]
Hello Thomas and Holger,

Sorry, it's my mistake! I misunderstood while reading the ticket #140 
I will provide new commit for this pull-request.

Thank you,
Son

**Review by test-fullautomation — APPROVED** [review]
Hi Son,
looks good to me now!
Thank you,
 Thomas

**Pull request merged**